### ISSUE TITLE
Create a custom password input component

### ISSUE THREAD

**Issue by RodriSanchez1** [cboard-org/cboard#1702]
Recently, we implemented the toggle password visibility feature on the password input in PR #1700. This was just for the Login form. I believe we need to use the same feature on the Sign Up form as well.

To achieve this, we can create a new functional component <PasswordField> and use it on both forms.

**Comment by bosiljkatodic00** [comment]
Hi! I am interested in working on this issue. Could you please assign it to me? Thanks!

**Comment by RodriSanchez1** [comment]
Yeah! For sure! @bosiljkatodic00  welcomee

**Comment by KirtiKamal** [comment]
hey @RodriSanchez1 I want to work on this issue can you please elaborate what kind of change you actually want to be done

**Comment by RodriSanchez1** [comment]
@bosiljkatodic00 are you working on this? If not let me know and I will assign @KirtiKamal 

A new `PasswordTextField` (or some descriptive name) component needs to be created as a funtional component and use it on Login and Sign Up forms.

We will wait until tomorrow to get the bosil response

**Comment by KirtiKamal** [comment]
Okay got it. You want me to create a PasswordTextField as a functional component that can be used in multiple file. And the pasword field only have one filed that is password correct me if i am wrong

**Comment by RodriSanchez1** [comment]
Yes! See the changes on the PR mencioned on this issue description

**Comment by KirtiKamal** [comment]
@RodriSanchez1 I have added PasswordTextField and given a PR kindly check

**Comment by bosiljkatodic00** [comment]
Hi @RodriSanchez1,

I apologize for the delay in my response. Please feel free to assign this issue to him, as I haven't had the opportunity to start working on it yet.

Thank you for your understanding.

**Comment by RodriSanchez1** [comment]
@bosiljkatodic00 that's okay! No problem at all. If you want to collaborate in other thing when you have free time, let me know.

@KirtiKamal cool! I will check it later!

**Comment by Zek1t0** [comment]
Hello, I’m interested in working on this issue. Could you confirm if it’s still available for someone to take? If you have any specific guidelines or suggestions for tackling this, please let me know. Thank you!
